_layouts/contact_us.html

```diff
@@ -10,26 +10,42 @@
   - Trusted Traveler Programs (Global Entry/Nexus/Sentri)
 ---
 
-{{ page.help_center_content | markdownify }} 
-
-<div class="page-content__section">
-  
-  {{ page.intro_content | markdownify }} 
-  
-  {% if site.contact_phone_number_enabled and site.contact_phone_number and 
-     site.contact_phone_number_location == 'top' %} 
-    {% include contact_phone.html %} 
+{% if site.contact_maintenance_start_time and site.contact_maintenance_end_time %}
+  <div id="contact-us-maintenance-alert" class="usa-alert usa-alert--warning" hidden>
+    <div class="usa-alert__body">
+      <p class="usa-alert__text">{{ page.maintenance_window_content }}</p>
+    </div>
+  </div>
+{% endif %}
+
+{{ page.help_center_content | markdownify }}
+
+<contact-us-form
+  maintenance-alert-id="contact-us-maintenance-alert"
+  maintenance-start-time="{{ site.contact_maintenance_start_time }}"
+  maintenance-end-time="{{ site.contact_maintenance_end_time }}"
+  class="page-content__section"
+>
+  {{ page.intro_content | markdownify }}
+
+  {% if site.contact_phone_number_enabled and site.contact_phone_number and
+     site.contact_phone_number_location == 'top' %}
+    {% include contact_phone.html %}
   {% endif %}
 
   <div class="desktop:grid-col-9">
     {% include contact_form.html common_applications=layout.common_applications %}
   </div>
 
   {% if site.contact_phone_number_enabled and site.contact_phone_number and
-     site.contact_phone_number_location == 'bottom' %} 
-    {% include contact_phone.html %} 
+     site.contact_phone_number_location == 'bottom' %}
+    {% include contact_phone.html %}
   {% endif %}
-</div>
+</contact-us-form>
 
 <div class="page-content__section">{{ page.partner_content | markdownify }}</div>
 <div class="page-content__section">{{ page.report_issue_content | markdownify }}</div>
+
+{% if site.contact_maintenance_start_time and site.contact_maintenance_end_time %}
+  <script src="{{ '/assets/js/contact_us_form_element.js' | relative_url }}"></script>
+{% endif %}
```

Comment:
Just to check my understanding, normally we'd add JS tags via `page.scripts`, but in this case, we don't want to even have the browser load the JS if the window isn't rendered?

Reply:
Correct. the thinking here was to prevent the script from being loaded if there is no planned maintenance window.

I'm on the fence whether we just load it anyways, to limit the number of potential variables, to ensure that `<contact-us-form>` always has a defined custom element implementation, and to support potential enhancements to the form implementation beyond just maintenance windows.

Reply:
I decided to leave this as-is for now. It won't be much trouble to load it unconditionally if we get to the point of enhancing the custom element further.